src/device_info.sh

```diff
@@ -202,31 +244,44 @@ function get_os()
   local os_id_like
 
   target=${target:-"${options_values['TARGET']}"}
-  flag=${flag:-'SILENT'}
```

Comment:
Is there a reason for deleting this line?

Reply:
same as my previous comment.